core/server/worker/src/test/java/alluxio/worker/grpc/BlockReadHandlerTest.java

```diff
@@ -30,16 +31,28 @@
 import io.netty.util.ResourceLeakDetector;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
+@RunWith(PowerMockRunner.class)
```

Comment:
if we remove `final` for `DefaultBlockWorker` , can we avoid `PowerMockRunner`?
PowerMock introduces a lot limitation and let's avoid to use it

Reply:
Add a TODO(lu) in the code and will give another PR to address the PowerMockRunner issue of the tests, thanks!

Reply:
I took a deeper look, DefaultBlockWorker is a complicated class with many other classes involved, it's not isolated like the previous BufferCachingDataReader class. The `BlockWorkerTest` itself is using PowerMock runner. Would suggest changing the whole `BlockWorkerTest` as well. `BlockReadHandlerTest` is a small extend of the `BlockWorkerTest`. This test may need more careful design and think through.